rails/client/package.json

```diff
@@ -17,7 +17,7 @@
     "client/node_modules"
   ],
   "dependencies": {
-    "antd": "^3.2.3",
+    "antd": "3.2.3",
```

Comment:
There was a non-passive change in one of the ant components that needs some more investigating, in the mean time I locked down the version we consume.